client/src/components/Dropdown/_Dropdown.scss

```diff
@@ -1,69 +1,85 @@
-.dropdown {
-  position: relative;
-
-  &__button {
-
-  }
-
-  &__circle {
-    display: grid;
-    justify-content: center;
-    align-content: center;
-
-    border-radius: 50%;
-    border: 1px solid $color-primary;
-    cursor: pointer;
-    height: 2rem;
-    position: relative;
-    width: 2rem;
-
-    & > .fa-ellipsis-h {
-      color: $color-primary;
-    }
-  }
-
-  &__plus {
-    display: grid;
-
-    color: white;
-
-    background-color: $color-primary;
-    border-radius: $border-radius;
-    cursor: pointer;
-    padding: 1rem;
-    width: 3.6rem;
-  }
-
-  &__list {
-    border: 2px solid $color-primary-light;
-    border-radius: $border-radius;
-    background-color: white;
-    opacity: 0;
-    margin-top: 0.5rem;
-    position: absolute;
-    min-width: 20rem;
-    transform: translateY(-3px);
-    transition: all 0.2s;
-    z-index: -1;
-
-    &.active {
-      display: block;
-      color: $color-primary;
-      opacity: 1;
-      transform: translateY(0);
-      z-index: 10;
-    }
-  }
-
-  &__item {
-    color: $color-primary-light;
-    font-size: 1.4rem;
-
-    cursor: pointer;
-    padding: 0.5rem 1rem;
-
-    &:hover {
-      background-color: $color-grey-1;
-    }
-  }
+.dropdown {

```

Comment:
Be sure to view this file with https://github.com/codefordenver/Comrad/pull/278/files?w=1, which ignores the whitespace changes.